cmd/podman/system/connection/remove.go

```diff
@@ -21,21 +21,42 @@ var (
 		Example: `podman system connection remove devl
   podman system connection rm devl`,
 	}
+
+	rmOpts = struct {
+		All bool
+	}{}
 )
 
 func init() {
 	registry.Commands = append(registry.Commands, registry.CliCommand{
 		Command: rmCmd,
 		Parent:  system.ConnectionCmd,
 	})
+
+	flags := rmCmd.Flags()
+	flags.BoolVarP(&rmOpts.All, "all", "a", false, "Remove all connections")
 }
 
-func rm(_ *cobra.Command, args []string) error {
+func rm(cmd *cobra.Command, args []string) error {
 	cfg, err := config.ReadCustomConfig()
 	if err != nil {
 		return err
 	}
 
+	if cmd.Flags().Changed("all") {
```

Comment:
I could say all==false.
I think you should just check for all==true.

Reply:
thanks added a check.